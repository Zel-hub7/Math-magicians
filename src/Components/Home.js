import './navBar.css';
const Home = ()=> {
    return(
        <div className="container">
           <h1>
            Welcome To Our Page
           </h1>
           <p>Dive into the world of numbers with our React-powered calculator website. Whether you're a math enthusiast or just looking for quick calculations, our user-friendly interface makes math a breeze. From simple sums to complex equations, Math Magician is your go-to tool for all things math. Start exploring today and unleash the magic of numbers! </p>
        </div>
    )
}
export default Home;