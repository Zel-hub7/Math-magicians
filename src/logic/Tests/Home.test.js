import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../Components/Home';

test('should match Home componet snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
