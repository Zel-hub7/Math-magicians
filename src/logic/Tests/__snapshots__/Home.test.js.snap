// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Home componet snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      Welcome To Our Page
    </h1>
    <p>
      Dive into the world of numbers with our React-powered calculator website. Whether you're a math enthusiast or just looking for quick calculations, our user-friendly interface makes math a breeze. From simple sums to complex equations, Math Magician is your go-to tool for all things math. Start exploring today and unleash the magic of numbers! 
    </p>
  </div>
</DocumentFragment>
`;
